### PR TITLE
react-helmet: Add missing props

### PR DESCRIPTION
**Pull request by kokjinsam**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/nfl/react-helmet/blob/master/src/Helmet.js>>

----

Some props were removed when the definitions were updated from v4 to v5.

### CODE REVIEW

File: types/react-helmet/index.d.ts [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-// Definitions by: Evan Bremer <https://github.com/evanbb>, Isman Usoh <https://github.com/isman-usoh>, François Nguyen <https://github.com/lith-light-g>
+// Definitions by: Evan Bremer <https://github.com/evanbb>, Isman Usoh <https://github.com/isman-usoh>, François Nguyen <https://github.com/lith-light-g>, Kok Sam <https://github.com/sammkj>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
 import * as React from "react";
 
 interface HelmetProps {
+    base?: any;
+    bodyAttributes: Object;
```
Comment (lith-light-g) [inline types/react-helmet/index.d.ts]: Should be optional

File: types/react-helmet/index.d.ts [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
     encodeSpecialCharacters?: boolean;
+    htmlAttributes?: any;
+    onChangeClientState?: (newState: any) => void;
+    link?: Array<any>;
+    meta?: Array<any>;
+    noscript?: Array<any>;
+    script?: Array<any>;
+    style?: Array<any>;
+    title?: string;
+    titleAttributes: Object;
```
Comment (lith-light-g) [inline types/react-helmet/index.d.ts]: Should be optional

**Review by phurytw — CHANGES_REQUESTED** [review]
I didn't realize the props were still used in v5 :sweat_smile:

I think all of the props should be optional so we can still use the Helmet component without any props (as seen in the [example](https://github.com/nfl/react-helmet#example)).

I'm pretty sure the tests would even pass if you set those props as optional.

File: types/react-helmet/index.d.ts (hunk 1 above)
Comment (sammkj) [inline types/react-helmet/index.d.ts]: I missed those. I will push an update now

**Review by phurytw — APPROVED** [review]

**Pull request merged**